rfcs/text/0000-create-file-elf.md

```diff
@@ -0,0 +1,306 @@
+          description: >
+            ELF Section List flags.
+          type: keyword
+        - name: name
+          description: >
+            ELF Section List name.
+          type: keyword
+        - name: physical_offset
+          description: >
+            ELF Section List offset.
+          type: keyword
+        - name: section_type
+          description: >
+            ELF Section List type.
+          type: keyword
+        - name: size
+          description: >
+            ELF Section List size.
+          format: bytes
+          type: long
+        - name: virtual_address
+          description: >
+            ELF Section List virtual address.
+          format: string
+          type: long
+    - name: exports
+      description: >
+        List of exported element names and types
+      release: beta
+      type: group
+      fields:
+        - name: name
+          description: >
+            Name of exported symbol
+          type: keyword
+          default_field: false
+        - name: type
+          description: >
+            Type of exported symbol
+          type: keyword
+          default_field: false
+    - name: imports
+      description: >
+        List of imported element names and types
+      release: beta
+      type: group
+      fields:
+        - name: name
+          description: >
+            Name of imported symbol
+          type: keyword
+          default_field: false
+        - name: type
+          description: >
+            Type of imported symbol
+          type: keyword
+          default_field: false
+    - name: shared_libraries
+      description: >
+        List of shared libraries used by this ELF object
+      type: keyword
+    - name: telfhash
+      description: >
+        telfhash hash for ELF files.
+      type: keyword
+    - name: flattened
+      default_field: false
+      description: >
+        Flattened ELF events from VirusTotal Intelligence Live Hunt results.
+      release: beta
+      type: group
+      fields:
+        - name: segment_list
+          description: >
+            ELF object segment list.
+          type: flattened
+```
+
+<!--
+Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+-->
+
+<!--
+Stage 3: Add more real world example source documents so we have at least 2 total, but ideally 3. Format as described in stage 2.
+-->
+
+## Scope of impact
+
+**Stage 2**
+
+There should be no breaking changes, depreciation strategies, or significant refactoring as this is creating a sub-field for the existing `file.` fieldset.
+
+While likely not a large-scale ECS project, there would be documentation updates needed to explain the new fields.
```

Comment:
We recently added the ability to create such [usage](https://github.com/elastic/ecs/tree/master/docs/usage#usage-docs) pages for each fieldset into the project. 😄 

The [user usage](https://www.elastic.co/guide/en/ecs/1.8/ecs-user-usage.html) doc is an example.

Reply:
Nice!

I added a start for `macho` and `elf`.